releng/org.eclipse.tracecompass.target/tracecompass-e4.36.target

```diff
@@ -49,6 +49,7 @@
 </location>
 <location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
 <unit id="org.mozilla.javascript" version="0.0.0"/>
+<unit id="com.fasterxml.jackson.core.jackson-core" version="0.0.0"/>
```

Comment:
This should be under the orbit-aggregation/release/4.32.0/ above instead of orbit-aggregation/release/4.29.0/ to get the version for e4.32 target. (line 47)

Reply:
fixed